src/main/java/com/feelmycode/parabole/dto/CouponCreateResponseDto.java

```diff
@@ -0,0 +1,14 @@
+package com.feelmycode.parabole.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.Getter;
+
+@Getter // dto 에 getter 안붙여주면 406 Not Acceptable 납니다
+@AllArgsConstructor
+public class CouponCreateResponseDto {
+    private String couponName;
```

Comment:
권장하지 않는 어노테이션입니다.
그리고 함수명 다음줄은 띄어주는게 좋습니다.
```suggestion
public class CouponCreateResponseDto {

    private String couponName;
```